### PR TITLE
Use [ngClass] to avoid conflicts with [class.x] bindings

### LINKED ISSUE #1650: Tabs - customClass bug
When using the [customClass] input, changes to it interfere with the active/disabled classes. This is due to binding both to class (through {{}} bindings) and [class.x]

This happens:

tabsbug

### PR DESCRIPTION
**Pull request by tiagoroldao**
Fixes #1650

### CODE REVIEW

**Comment by valorkin** [comment]
simple and great, can you add simple test to cover #1650 issue?

File: src/tabs/tabset.component.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { TabsetConfig } from './tabset.config';
   selector: 'tabset',
   template: `
     <ul class="nav" [ngClass]="classMap" (click)="$event.preventDefault()">
-        <li *ngFor="let tabz of tabs" class="nav-item {{tabz.customClass}}"
+        <li *ngFor="let tabz of tabs" [ngClass]="'nav-item ' + tabz.customClass"
```
Comment (Le0Michine) [inline src/tabs/tabset.component.ts]: it can be:
`[ngClass]="['nav-item', tabz.customClass]"`

just to avoid concatenation in the template

Comment (tiagoroldao) [reply src/tabs/tabset.component.ts]: True, and I forgot to handle undefined values as well.

**Comment by tiagoroldao** [comment]
@valorkin  tests added 👍 

Changed `expectActiveTabs` to use a `getTabItems` function as to break on my tests before this fix (rightfully), and because the style changes are due to `.nav-item.active`, not `.nav-link.active`.

**Review by valorkin — APPROVED** [review]
Awesome, thanks

**Pull request merged**